components/SelectComponent.tsx

```diff
@@ -0,0 +1,99 @@
+import { fhirJson } from "@fhir-typescript/r4-core";
+import { useState, useEffect, Dispatch, SetStateAction } from "react";
+import { Autocomplete } from "@mantine/core";
+
+/**
+ * @param props include the string resourceType
+ * @returns a component with a loading component, server error, or autocomplete select component populated with resource IDs
+ */
+export default function SelectComponent(props: {
+  resourceType: string;
+  setPractitionerValue: Dispatch<SetStateAction<string>>;
+  practitionerValue: string;
+}) {
+  const resourceType = props.resourceType;
+  const [pageBody, setPageBody] = useState<fhirJson.Bundle>();
```

Comment:
I think pageBody doesn't quite capture what this variable is storing. `responseBody` may be more appropriate. Also v open to suggestions